metrics/influx.go

```diff
@@ -0,0 +1,91 @@
+package metrics
+
+import (
+	"bytes"
+	"time"
+
+	"github.com/containous/traefik/log"
+	"github.com/containous/traefik/safe"
+	"github.com/containous/traefik/types"
+	kitlog "github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/metrics/influx"
+	influxdb "github.com/influxdata/influxdb/client/v2"
+)
+
+var influxClient = influx.New(map[string]string{}, influxdb.BatchPointsConfig{}, kitlog.LoggerFunc(func(keyvals ...interface{}) error {
+	log.Info(keyvals)
+	return nil
+}))
+
+type bufWriter struct {
```

Comment:
Could you explain why you created `bufWriter`?

Reply:
https://github.com/go-kit/kit/blob/master/metrics/influx/influx.go#L99

Reply:
We can call it with different name to avoid naming confusion

Reply:
yes please